### PR TITLE
fix(themes): less3/4 needs font import interpreted as raw css

### PR DESCRIPTION
**Pull request by lubber-de**
## Description

When using webpack 4+ and LESS 3/4 the font imports have to be interpreted as raw CSS, otherwise the build breaks.
This is already fixed for the default theme since 3 years, but any of the included optional themes will fail.

I also updated the google font api usage to v2

### CODE REVIEW

File: src/themes/material/modules/dropdown.overrides [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import url(https://fonts.googleapis.com/css?family=Roboto:400,700);
+@import (css) url('https://fonts.googleapis.com/css2?family=Roboto');
```
Comment (exoego) [inline src/themes/material/modules/dropdown.overrides:1]: Forgot [weight range](https://developers.google.com/fonts/docs/css2#axis_ranges) ? 
```suggestion
@import (css) url('https://fonts.googleapis.com/css2?family=Roboto:wght@400..700');
```

File: src/themes/instagram/views/card.overrides [hunk 2]
```diff
@@ -3,7 +3,7 @@
 *******************************/
 
 
-@import url(https://fonts.googleapis.com/css?family=Montserrat:700,400);
+@import (css) url('https://fonts.googleapis.com/css2?family=Montserrat');
```
Comment (exoego) [inline src/themes/instagram/views/card.overrides:6]: Forgot [weight range](https://developers.google.com/fonts/docs/css2#axis_ranges) ? 
```suggestion
@import (css) url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700,400');
```

File: src/themes/material/modules/dropdown.overrides (hunk 1 above)
Comment (lubber-de) [reply src/themes/material/modules/dropdown.overrides:1]: Only weight 400 is available in the v2 api, that's why i removed the weight completely.
Your suggestions results in an unresolvable link:
https://fonts.googleapis.com/css2?family=Roboto:wght@400..700
image

File: src/themes/instagram/views/card.overrides (hunk 2 above)
Comment (lubber-de) [reply src/themes/instagram/views/card.overrides:6]: Same here: only 400 is available and your suggestion results in an unresolved link
https://fonts.googleapis.com/css2?family=Montserrat:wght@700,400
image

**Review by exoego — APPROVED** [review]
LGTM

**Pull request merged**